constellation/constellation.py

```diff
@@ -87,7 +87,8 @@ class ConstellationContainer:
 
     def __init__(self, name, image, args=None,
                  mounts=None, ports=None, environment=None, configure=None,
-                 entrypoint=None, working_dir=None, labels=None):
+                 entrypoint=None, working_dir=None, labels=None,
+                 preconfigure=None, network="none"):
```

Comment:
What is up with this network change? Seems we'd not want to have things on the none network by default?

Reply:
Ah, I see, this is to deal with the odd dance below. Is the `__init__` method ever not given `"none"` here? could  just set that to "none" below?

Reply:
so I had to change this because of the wodin proxy, I need to specify the network for it and not use this "none" network trick because it would sometimes start and fail to find one of the wodin sites (since they have already been disconnected from the "none" network in that faff below), we could sleep the proxy for a bit but this is just a more direct way I think